components/progress-bar/src/ProgressBar.js

```diff
@@ -0,0 +1,90 @@
+/** @jsx jsx */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { jsx, useTheme } from '@westpac/core';
+
+import { SrOnly } from '../../accessibility-helpers/src';
+
+// ==============================
+// Utils
+// ==============================
+
+// ==============================
+// Component
+// ==============================
+
+/**
+ * Progress Bar: A visual indication of progress. Use when loading content or to indicate how far along the user is in a journey.
+ */
+export const ProgressBar = ({ valuemin, valuemax, valuenow, skinny, ...props }) => {
+	const { progressBar } = useTheme();
+
+	// Common styling
+	const common = {
+		height: skinny ? 10 : 24,
+		marginBottom: progressBar.marginBottom,
+		overflow: 'hidden',
+		backgroundColor: 'white',
+		borderRadius: 1000,
```

Comment:
I don't know why this needs to be 1000... Do you?

Can't we just set to `height / 2`?

Reply:
No idea... I changed it to its height...